components/layout/layout.tsx

```diff
@@ -22,10 +22,9 @@ const Layout = ({ children }: LayoutProps) => {
       <style>{`
                 .layout__container {
```

Comment:
Please clean the class of this html so only one ".layout__container" will stay.

Reply:
> @NisanAyash
> Great job!! And well done on the sass loop challenge 💪🏻
> Just one problem, this is how it is look between 1420-1459 view width, and a few suggestion on the code :)
> 
> ![image](https://user-images.githubusercontent.com/47606575/113537617-e2c9fa80-95e1-11eb-968a-5bcf0e811429.png)

Hey, @MichalPorag Thanks for your comments...
I'll fix that and send PR again soon.